tutorials/rdf_fitting_demo.ipynb

```diff
@@ -1,29 +1,76 @@
 {
```

Comment:
<p>This is great! I'm going to do some testing with scipy.find_peaks and see how it performs on the test examples I created. If it outperforms the current implementation, I'll use it as the default.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/solvation-analysis/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>